sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/OpenAIAsyncClient.java

```diff
@@ -1445,34 +1453,6 @@ public Mono<Response<BinaryData>> getAudioTranslationAsPlainTextWithResponse(
                 deploymentOrModelName, audioTranslationOptions, requestOptions);
     }
 
-    /**
-     * Gets transcribed text and associated metadata from provided spoken audio data. Audio will be transcribed in the
-     * written language corresponding to the language it was spoken in.
-     *
-     * @param deploymentOrModelName Specifies either the model deployment name (when using Azure OpenAI) or model name
-     *     (when using non-Azure OpenAI) to use for this request.
-     * @param audioTranscriptionOptions The configuration information for an audio transcription request.
-     * @throws IllegalArgumentException thrown if parameters fail the validation.
-     * @throws HttpResponseException thrown if the request is rejected by server.
-     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
-     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
-     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
-     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
-     * @return transcribed text and associated metadata from provided spoken audio data on successful completion of
-     *     {@link Mono}.
-     */
-    @Generated
-    @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<AudioTranscription> getAudioTranscriptionAsResponseObject(
-            String deploymentOrModelName, AudioTranscriptionOptions audioTranscriptionOptions) {
-        // Generated convenience method for getAudioTranscriptionAsResponseObjectWithResponse
-        RequestOptions requestOptions = new RequestOptions();
-        return getAudioTranscriptionAsResponseObjectWithResponse(
-                        deploymentOrModelName, BinaryData.fromObject(audioTranscriptionOptions), requestOptions)
-                .flatMap(FluxUtil::toMono)
-                .map(protocolMethodData -> protocolMethodData.toObject(AudioTranscription.class));
-    }
```

Comment:
These APIs are removed when generating code, as it won't work with `multipart/form-data`. Customer should use customized API `getAudioTranslation` or `getAudioTranslationWithResponse`.

Reply:
However, there seems to be a bug about `getAudioTranscriptionAsPlainText`, which should also be a `multipart/form-data`

Reply:
Yeah. I believe for they are not working for those generated audio translations and transcriptions API

Reply:
0.9.0 codegen automatically hide generated API from "multipart/form-data" (convenience API would be removed, protocol API would be hidden -- "package private", hence this change).

Reply:
> We also have the generated APIs below that can be removed. Should
> 
> getAudioTranscriptionAsPlainText getAudioTranscriptionAsPlainTextWithResponse
> 
> getAudioTranslationAsPlainText getAudioTranslationAsPlainTextWithResponse getAudioTranslationAsResponseObjectWithResponse

`getAudioTranslationAsResponseObjectWithResponse` already removed as 0.9.0 (next block of removal in this PR)

That is due to a bug mentioned in https://github.com/Azure/azure-sdk-for-java/pull/37179#discussion_r1357946496

The `MultipartFormDataRequestHeadersTraits` seems not having effect on
```ts
op getAudioTranscriptionAsPlainText is Azure.Core.Foundations.ResourceOperation<
  Deployment,
  AudioTranscriptionOptions,  // response_format must be one of: text, srt, vtt
  string,
  MultipartFormDataRequestHeadersTraits
>;
```

So TypeSpec give this to typespec-java not as a "multipart/form-data".

Do you want me to log an issue to spec repo (for Jose Alvarez to coordinate)?

Reply:
My bad. `getAudioTranslationAsResponseObjectWithResponse` is no longer a public API as it shows in the PR. I wasn't aware of it.

`Do you want me to log an issue to spec repo (for Jose Alvarez to coordinate)?`
Yes, please. It helps to not explore the non-working public API to customers.

Reply:
https://github.com/Azure/azure-rest-api-specs/issues/26299